TAS_Engine/Convert/Environment_oM/InternalGain.cs

```diff
@@ -40,60 +41,89 @@ public static partial class Convert
         [Description("BH.Engine.TAS.Convert ToBHoM => gets BH.oM.Environment.Elements.InternalGain from TAS TBD InternalGain")]
         [Input("tbdInternalGain", "TAS TBD InternalGain")]
         [Output("BHoM Environmental InternalGain object")]
-        public static BHE.InternalGain ToBHoM(this TBD.InternalGain tbdInternalGain)
+        public static List<BHE.Gain> ToBHoM(this TBD.InternalGain tbdInternalGain)
         {
             if (tbdInternalGain == null) return null;
 
-            BHE.InternalGain internalGain = new BHE.InternalGain();
-            internalGain.Name = tbdInternalGain.name;
-            internalGain.Illuminance = tbdInternalGain.targetIlluminance;
-            internalGain.OutsideAirRatePerPerson = tbdInternalGain.freshAirRate;
-            internalGain.PersonGain = tbdInternalGain.personGain;
-            internalGain.RadiationProperties.EquipmentRadiation = tbdInternalGain.equipmentRadProp;
-            internalGain.RadiationProperties.LightingRadiation = tbdInternalGain.lightingRadProp;
-            internalGain.RadiationProperties.OccupantRadiation = tbdInternalGain.occupantRadProp;
-            internalGain.CoefficientProperties.EquipmentViewCoefficient = tbdInternalGain.equipmentViewCoefficient;
-            internalGain.CoefficientProperties.LightingViewCoefficient = tbdInternalGain.lightingViewCoefficient;
-            internalGain.CoefficientProperties.OccupantViewCoefficient = tbdInternalGain.occupantViewCoefficient;
-
-            internalGain.Profiles = tbdInternalGain.Profiles();
+            List<BHE.Gain> gains = new List<BHE.Gain>();
 
             Dictionary<string, object> tasData = new Dictionary<string, object>();
             tasData.Add("InternalGainActivityID", tbdInternalGain.activityID);
             tasData.Add("InternalGainDescription", tbdInternalGain.description);
             tasData.Add("InternalDomesticHotWater", tbdInternalGain.domesticHotWater);
 
-            internalGain.CustomData = tasData;
-            return internalGain;
+            BHE.Gain occupantGain = new BHE.Gain();
+            occupantGain.Name = tbdInternalGain.name;
+
+            BHP.GainPropertiesPeople peopleGain = new BHP.GainPropertiesPeople();
+            peopleGain.GainType = BHE.GainType.People;
+            peopleGain.GainUnit = BHE.GainUnit.NumberOfPeople;
```

Comment:
please see comments about metabolic rate, this will be method to calculate Number Of people